client/my-sites/hosting/sftp-card/index.js

```diff
@@ -88,14 +125,18 @@ const SFTPCard = ( { translate, username, password, errorCode, siteId, loading }
 
 export default connect( state => {
 	const siteId = getSelectedSiteId( state );
-	const sftpDetails = requestAtomicSFTPDetails( siteId );
-	const username = get( sftpDetails, 'data.username', null );
+
+	// @TODO dummy data is added here and in component state; remove when API is merged
+	// const sftpDetails = requestAtomicSFTPDetails( siteId );
+
+	// const username = get( sftpDetails, 'data.username', null );
+	// const errorCode = get( sftpDetails, 'error.status', null );
 
 	return {
 		siteId,
-		username,
-		password: get( sftpDetails, 'data.password', null ),
-		errorCode: get( sftpDetails, 'error.status', null ),
-		loading: get( sftpDetails, 'status', null ) === 'pending' || ! username,
+		// username,
+		// noSftpUser: errorCode === 404,
+		// password: get( sftpDetails, 'data.password', null ),
+		// loading: sftpDetails.status === 'pending' || ! username,
```

Comment:
It's fine to simply delete this. We have past changes in git, if we need to refer to former changes.